qemu/tests/chardev_acpi.cfg

```diff
@@ -0,0 +1,5 @@
+- chardev_acpi:
+    only Linux 
+    no q35
```

Comment:
`no q35` means many other machine type, i guess you mean `only i440fx` 
@nanliu-r  please help confirm this, thanks!

Reply:
> `no q35` means many other machine type, i guess you mean `only i440fx`
> @nanliu-r please help confirm this, thanks!

we don't limit the machine type for this case.

Reply:
@michiboo  we do not limit the machine type, please remove 'no q35'